backends/include/dppl_sycl_queue_interface.hpp

```diff
@@ -0,0 +1,209 @@
+//===--- dppl_sycl_queue_interface.hpp - DPPL-SYCL interface ---*- C++ -*---===//
+//
+//               Python Data Parallel Processing Library (PyDPPL)
+//
+// Copyright 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file contains the wrapper functions to manage SYCL queue objects.
+///
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "dppl_macros.hpp"
+#include <cstdlib>
+
+namespace dppl
+{
+
+/*!
+ * @brief Redefinition of Sycl's device_type so that we do not have to include
+ * sycl.hpp here and in the Python bindings.
+ *
+ */
```

Comment:
Added comments to the file. The issue I want to avoid is having to build the Cython extension using `dpcpp`. It can be problematic on Windows where Python is built using MSVC.